modules/network-observability-flowcollector-about.adoc

```diff
@@ -0,0 +1,121 @@
+* `ebpf.cacheMaxFlows` is a string that specifies the maximum duration that flows are aggregated in the eBPF agent before being forwarded to the processor. The higher the `cacheMaxFlows` value, the less traffic the eBPF agent generates, which reduces resource demands; however note that memory consumption is higher which may create latency in the flow collection.
+* `ebpf.cacheActiveTimeout` specifies the maximum number of flows that can be kept in the eBPF agent memory. If the number of flows being aggregated reaches this number, they are flushed to the Flow Processor, even if the `cacheActiveTimeout` period is not reached.The higher the `cacheActiveTimeout` value, the less traffic the eBPF agent generates, which reduces resource demands. 
+* `ebpf.interfaces` and `ebpf.excludeInterfaces` specify the interface names from where flows are collected. You can reduce the overall observed traffic by restricting or excluding interfaces, which can help with resrouce management. If empty, the agent will fetch all the interfaces in the system, excepting the ones listed in ExcludeInterfaces. If an entry is enclosed by slashes (e.g. `/br-/`), it will match as regular expression, otherwise it will be matched as a case-sensitive string.
+* `ebpf.debug.env` specifies custom environment variables to the
+Flowlogs-Pipeline pod. This field is useful for passing
+some concrete performance-tuning options, such as GOGC and
+GOMAXPROCS.
+
+
+|`spec.consolePlugin.qi`
+|`integer`
+|Specifies configuration settings related to the Loki client. The `flowlogs-pipeline` processor uses these configurations to connect and send data to Loki for storage. The following Loki setings can be specified:
+
+* `loki.batchWait` and `loki.batchSize control the batching mechanism, or how often data is sent to Loki. Similar to the eBPF agent batching, higher values will generate fewer traffic and consume less CPU, however it will increase a bit the memory consumption of the flowlogs-pipeline, which might increase collection latency.
+* `spec.loki.minBackoff`, `spec.loki.maxBackoff` and `spec.loki.maxRetries` control the retry mechanism. Retries might happen when Loki is unreachable or when it returns errors. Often, it is due to the rate limits configured on Loki server. When such situation occurs, it might not always be the best solution to increase rate limits (on server configuration side) or to increase retries. Increasing rate limits will put more pressure on Loki, so expect more memory and CPU usage, and also more traffic. Increasing retries will put more pressure on flowlogs-pipeline, as it will retain data for longer and accumulate more flows to send. When all the retry attempts fail, flows are simply dropped. Flow drops are counted in the metric netobserv_loki_dropped_entries_total
+
+|`spec.consolePlugin.quickFilters`
+|`string`
```

Comment:
this is a structured object

Reply:
Addressed here and elsewhere. I need to go back and make sure I note the bulleted configs but the row-level configs do have the correct now I think.